### PR TITLE
kraken: Manage contributor

### PR DESCRIPTION
**Pull request by azime**
. Jira : http://jira.canaltp.fr/browse/NAVP-328
. contributor added in disruption object.
. the new json of disruption is:

``` json
{
    "status": "active",
    "disruption_id": "05b2d960-612f-11e5-b5a0-00249b0dc60b",
    "severity": 
{
    "color": "#FF4455",
    "priority": 1,
    "name": "test A",
    "effect": "MODIFIED_SERVICE"
},
"impact_id": "2f0fcaf2-612f-11e5-b5a0-00249b0dc60b",
"application_periods":
[
    {
        "begin": "20150922T153700",
        "end": "20151122T163659"
    }
],
"messages":
[
{
    "text": "test RER B",
    "channel": 
{
    "content_type": "text/plain",
    "id": "31398f9e-5c6f-11e5-a82a-00249b0dc60b",
    "types": 
        [
            "sms"
        ],
        "name": "Message SMS "
    }
},
{
    "text": "test RER B",
    "channel": 
{
    "content_type": "text/plain",
    "id": "27ac9dc2-5c6f-11e5-a82a-00249b0dc60b",
    "types": 
        [
            "web"
        ],
        "name": "Message WEB"
    }
},
{
    "text": "test RER B",
    "channel": 
{
    "content_type": "text/plain",
    "id": "3c182740-5c6f-11e5-a82a-00249b0dc60b",
    "types": 
                [
                    "web"
                ],
                "name": "Titre "
            }
        }
    ],
    "updated_at": "19700101T010000",
    "uri": "2f0fcaf2-612f-11e5-b5a0-00249b0dc60b",
    "disruption_uri": "05b2d960-612f-11e5-b5a0-00249b0dc60b",
    "contributor": "shortterm.tr_transilien",
    "cause": "test communication",
    "id": "2f0fcaf2-612f-11e5-b5a0-00249b0dc60b"
}
```

### CODE REVIEW

File: source/jormungandr/tests/disruptions_tests.py [hunk 1]
```diff
@@ -85,6 +86,9 @@ def test_disruption_end_point(self):
         eq_(network_disrupt[0]['disruption_id'], 'disruption_on_line_A')
         eq_(network_disrupt[0]['uri'], 'too_bad_again')
 
+
+        eq_(hasattr(network_disrupt[1], 'contributor'), False)
```
Comment (antoine-de) [inline source/jormungandr/tests/disruptions_tests.py]: hasattr works for `dict` too ?

I would have done `'contributor' not it network_disrupt[1]`

File: source/chaos-proto [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit d6101fdb64b2acd118211986d466c594c3586313
+Subproject commit cfe098b73618a0c28dd8c350a1d2abbb7ad7a6b4
```
Comment (pbougue) [inline source/chaos-proto:1]: ok

File: source/navitia-proto [hunk 3]
```diff
@@ -1 +1 @@
-Subproject commit dbb0516153c314e309569f8a265d8747d4df3827
+Subproject commit 485a92b929d8e6d164592a2f4cadf2fb23c6bb82
```
Comment (pbougue) [inline source/navitia-proto:1]: ok

**Comment by xlqian** [comment]
except for antoine's comment, ok for me

**Comment by antoine-de** [comment]
@azime you need to rebase your branch (the data version number must be in conflict)

**Comment by pbougue** [comment]
The data version number is 46 on the current dev (I used a different version number from this PR on my PR), so it should be 47 on this one (or 46 at worse)

**Comment by azime** [comment]
corrections

**Pull request merged**